website/content/docs/nia/enterprise/license.mdx

```diff
@@ -27,6 +27,16 @@ The options to set the license in order of precedence:
 
 Note: the [options to set the license and order of precedence](/docs/enterprise/license/overview#binaries-without-built-in-licenses) is the same as Consul Enterprise server agents.
 
+### Updating the License
+
+After running Consul-Terraform-Sync Enterprise, your license will eventually approach expiration, and you'll want to update to a new license. To do so, have the new license value at hand or saved in a file to disk.
+
+Steps to update to a new license:
+1. Update the license environment variable or configuration that you used to set the prior license with the new license value or file path
+1. Stop and restart Consul-Terraform-Sync Enterprise
```

Comment:
```suggestion
You can set the Consul-Terraform-Sync Enterprise license by either pointing to a license file in your Consul configuration or by specifying the license key in an environment variable. Use the following procedure to update the license when it expires or is near the expiration date: 
1. Update the license environment variable with the new license value or point to the updated license file in your Consul configuration.
1. Stop and restart Consul-Terraform-Sync Enterprise.
```
Please ignore this suggestion if this isn't quite right or doesn't seem like an improvement. I was just trying to see if I could smooth out the wording a bit.

Reply:
I think this is an improvement to the language. I ended up doing a combination to use your improvements and clarify some details about the configuration and relation to other content (not in this PR). I appreciate the suggestion. Thank you